src/Installer/Stable.htm

```diff
@@ -1,6 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <html>
   <div class="releasenotes">
+    <h2>3.0.1 05-Mar-2018</h2>
```

Comment:
Not that this matters much, but it should probably be the Beta, or the Alpha .htm which was targeted by the script here.

Reply:
I will re do that, then. Should it be Beta or Alpha?

Reply:
Lets use Beta.